src/sentry/relay/config/metric_extraction.py

```diff
@@ -406,11 +406,12 @@ def convert_widget_query_to_metric(
     """
     metrics_specs: list[HashedMetricSpec] = []
 
-    if not widget_query.aggregates:
```

Comment:
@k-fish as part of this PR, it would be good to see if this can be handled:
https://github.com/getsentry/sentry/pull/65700/files#r1500845435

> ...when you do the tables w/ aggregates change, those aggregates are inside columns not aggregates, so I was asking if just like how we also check columns for columns that are functions there if we need to do the same thing here